mmdet/models/utils/res_layer.py

```diff
@@ -0,0 +1,79 @@
+from torch import nn as nn
+
+from mmdet.ops import build_conv_layer, build_norm_layer
+
+
+class ResLayer(nn.Sequential):
+    """ResLayer to build ResNet style backbone.
+
+    Args:
+        block (nn.Module): block used to build ResLayer
+        inplanes (int): inplanes of block
+        planes (int): planes of block
+        num_blocks (int): number of blocks
+        stride (int): stride of the first block
```

Comment:
For arguments with a default value, use the following description: `Stride of the first block. Default: 1.`